java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -464,14 +491,15 @@ private void logState(MethodInvocationTree mit) {
 
   private void debugPrint(Object... toPrint) {
     if (DEBUG_MODE_ACTIVATED) {
-      LOG.error(Joiner.on(" - ").join(toPrint));
+      LOG.error(Joiner.on(" - ").join(toPrint)+"   "+workList.size());
     }
   }
 
-  public void enqueue(ExplodedGraph.ProgramPoint programPoint, ProgramState programState) {
+  public void enqueue(ExplodedGraph.ProgramPoint programPoint, ProgramState ps) {
+    ProgramState programState = removeUnreachableSymbolicValues(ps);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "programState" which hides the field declared at line 86. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)